db/data_migrate/20220330114035_add_cp_to_list.rb

```diff
@@ -0,0 +1,11 @@
+class AddCpToList < ActiveRecord::Migration[7.0]
+  def change
+    Organization.create!(
+      name: "Capital Placement",
+      pegged: true,
+      great_place_to_work: true,
```

Comment:
`great_place_to_work` this flag is used to determine whether the company is listed in https://greatplacetowork.lk/

It looks like `Capital Placement` was not part of Great place to work survey. So is it possible for you to remove `great_place_to_work: true` or make it `great_place_to_work: false`

Reply:
Sure, on it. Thanks, Dilum.